ifrs17/Utils/ActivityLog.ipynb

```diff
@@ -544,10 +544,10 @@
                 "\n    {",
                 "\n        var exportResult = await Builder.ExecuteAsync();",
                 "\n        var exportFile = await (new ExportFile(Builder, ImportVariable, Session)).InitializeExportDataAsync();",
-                "\n        var activity = new ImportExportActivity(exportResult.ActivityLog, Session) with {Category = \"Export to File\", ",
+                "\n        var activity = new Activity(exportResult.ActivityLog, Session) with {Category = \"Export to File\", ",
                 "\n                                                                                    SourceId = exportFile.Id};",
-                "\n        await DataSource.UpdateAsync<ExportFile>(exportFile.RepeatOnce());",
-                "\n        await DataSource.UpdateAsync<ImportExportActivity>(activity.RepeatOnce());",
+                "\n        await DataSource.UpdateAsync<IOContent>(exportFile.RepeatOnce());",
```

Comment:
I would like to point out that I chose to replace `ExportFile` with `IOContent` (ex `KeyedImportExport`) since there doesn't exist an `ExportFile` table in the DB anymore after the latest changes and the exported files were not tracked otherwise (FYI @dcolleoni).